lib/src/constants/text_style.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class TextStylesApp {
```

Comment:
I agree with Mr.Hai that you may need to use an app theme then use Theme.of(context).textTheme..copyWith.
You can take a look at Tam_Binh_Hai repo for this kind of implementation.

Reply:
I have already added app_theme